src/components/AssetsComponent.jsx

```diff
@@ -43,6 +43,7 @@ export class AssetsComponent extends Component {
           <div id="page-heading-section">
             <Header as="h1" id="page-headings" floated="left" content="Assets List" />
             <Divider id="assets-divider" />
+            <FilterButton render={() => {}} />

```

Comment:
Is the empty `render` prop really necessary? It is doing nothing in this case, apart from instantiating the function on each render.

If the `render` prop is required in the `FilterButton` component, then use a default prop in the actual component instead of providing the default here. That is:

In this component, you'll just have
```
<FilterButton />
```

Then in `FilterButton.jsx` you'll have:
```
FilterButton.defaultProps = {
  render: () => {}
};
```